.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+  "endOfLine": "lf",
+  "printWidth": 120,
+  "semi": true,
+  "singleQuote": true,
+  "tabWidth": 2,
+  "trailingComma": "all",
+  "useTabs": false
+}
```

Comment:
`endOfLine`, `semi`, `tabWidth` and `useTabs` could be omitted as the selected settings match the [defaults](https://prettier.io/docs/en/options.html) already

Reply:
That is a good point, and one I generally hew towards. With the codebase having a mix of all of these things (sometimes in the same file) I leaned toward being explicit instead of implicit for clarity.  This way when each file is formatted as we add things down the line, it's still clear where the rules are coming from.

Reply:
Oh, you're right! Explicit beats implicit 👍